docs/resources/sql_table.md

```diff
@@ -0,0 +1,91 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_sql_table (Resource)
+
+Within a metastore, Unity Catalog provides a 3-level namespace for organizing data: Catalogs, databases (also called schemas), and tables / views.
+
+A `databricks_sql_table` is contained within [databricks_schema](schema.md).
+
+## Example Usage
+
+```hcl
+resource "databricks_catalog" "sandbox" {
+  metastore_id = databricks_metastore.this.id
+  name         = "sandbox"
+  comment      = "this catalog is managed by terraform"
+  properties = {
+    purpose = "testing"
+  }
+}
+
+resource "databricks_schema" "things" {
+  catalog_name = databricks_catalog.sandbox.id
+  name         = "things"
+  comment      = "this database is managed by terraform"
+  properties = {
+    kind = "various"
+  }
+}
+
+resource "databricks_sql_table" "thing" {
+  provider           = databricks.workspace
+  name               = "quickstart_table"
+  catalog_name       = databricks_catalog.sandbox.id
+  schema_name        = databricks_schema.things.id
+  table_type         = "MANAGED"
+  data_source_format = "DELTA"
+  storage_location   = ""
+  column {
+    name      = "id"
+    position  = 0
```

Comment:
do we want to infer the position of the column from terraform list or do we want to expose `position` argument in the column block?

Reply:
I was leaning the same way, current state is a bit verbose.

Reply:
position is now removed